### PR TITLE
chore: remove circular dependencies

### PR DESCRIPTION
**Pull request by theoboldfrazier**
## Description

Due to an elusive issue on iPhone, we noticed that we have about 110 circular dependencies. We need to remove these circular dependencies. This is hoping that we can fix the issue on iPhone, but this needs to happen.

## Notice

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk

There are a lot of files touched with this PR. Though each individual change is fairly low risk, there is a risk of regressions in this PR.

## Testing

A full regression test would be needed to catch any one off issues.

### CODE REVIEW

File: src/components/Delegate/StakingOpportunitiesRow.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Flex, HStack } from '@chakra-ui/layout'
 import { Button, Skeleton, SkeletonCircle } from '@chakra-ui/react'
 import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
-import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { StakingAction } from 'plugins/cosmos/components/modals/Staking/StakingTypes'
```
Comment (0xdef1cafe) [inline src/components/Delegate/StakingOpportunitiesRow.tsx:4]:
```suggestion
import { StakingAction } from 'plugins/cosmos/components/modals/Staking/StakingCommon'
```

File: src/components/Modals/Send/AddressInput/AddressInput.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { useHistory } from 'react-router-dom'
 import { QRCodeIcon } from 'components/Icons/QRCode'
 
 import { SendFormFields, SendInput } from '../Form'
-import { SendRoutes } from '../Send'
+import { SendRoutes } from '../SendTypes'
```
Comment (0xdef1cafe) [inline src/components/Modals/Send/AddressInput/AddressInput.tsx:8]:
```suggestion
import { SendRoutes } from '../SendCommon'
```

File: src/components/Modals/Send/hooks/useSendDetails/useSendDetails.test.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { useChainAdapters } from 'context/PluginProvider/PluginProvider'
 import { useWallet } from 'context/WalletProvider/WalletProvider'
 import { ensLookup } from 'lib/ens'
 import { fromBaseUnit } from 'lib/math'
-import { PortfolioBalancesById } from 'state/slices/portfolioSlice/portfolioSlice'
+import { PortfolioBalancesById } from 'state/slices/portfolioSlice/portfolioSliceTypes'
```
Comment (0xdef1cafe) [inline src/components/Modals/Send/hooks/useSendDetails/useSendDetails.test.tsx:12]:
```suggestion
import { PortfolioBalancesById } from 'state/slices/portfolioSlice/portfolioSliceCommon'
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
rename all "fooTypes" to "fooCommon"

**Review by 0xdef1cafe — APPROVED** [review]

**Review by cjthompson — APPROVED** [review]

**Pull request merged**